src/main/java/seedu/address/logic/commands/InvokeCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.InvokeCommandException;
+import seedu.address.logic.parser.CommandRecorder;
+import seedu.address.model.Model;
+
+/**
+ * Invoke the most recent command that was typed in by user.
+ * Invoke will not count "invoke" as a result, it will return the most recent command that is not "invoke".
+ */
+public class InvokeCommand extends Command {
+
+    public static final String COMMAND_WORD = "invoke";
```

Comment:
I feel that "invoke" is not a good representation of this command. Perhaps "history" would be more suitable?

Reply:
But the COMMAND_WORD of every other commands are exactly same as the command name... I wanted to keep the naming style consistent.

Reply:
Maybe the command itself could be changed to `history`, what do you think?

Reply:
OK